### PR TITLE
Job to push cluster operator images

### PR DESCRIPTION
**Pull request by csrwng**
Builds cluster operator images and (for now) pushes them to the cluster-operator namespace on api.ci 
This is meant to run as a post-commit job.

### CODE REVIEW

File: sjb/generated/push_cluster_operator_images.xml [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+docker tag clusterapi:latest registry.svc.ci.openshift.org/openshift-cluster-operator/kubernetes-cluster-api:latest
+docker push registry.svc.ci.openshift.org/openshift-cluster-operator/kubernetes-cluster-api:latest
+SCRIPT
+chmod +x &#34;${script}&#34;
+scp -F ${WORKSPACE}/.config/origin-ci-tool/inventory/.ssh_config &#34;${script}&#34; openshiftdevel:&#34;${script}&#34;
+ssh -F ${WORKSPACE}/.config/origin-ci-tool/inventory/.ssh_config -t openshiftdevel &#34;bash -l -c \&#34;timeout 7200 ${script}\&#34;&#34; </command>
+        </hudson.tasks.Shell>
+  </builders>
+  <publishers>
+    <org.jenkinsci.plugins.postbuildscript.PostBuildScript plugin="postbuildscript@0.17">
```
Comment (dobbymoodge) [inline sjb/generated/push_cluster_operator_images.xml]: Rebase and regenerate this xml, the postbuildscript has breaking changes that are in the updated template.

Comment (csrwng) [reply sjb/generated/push_cluster_operator_images.xml]: thx will do

Comment (csrwng) [reply sjb/generated/push_cluster_operator_images.xml]: done

**Comment by dobbymoodge** [comment]
lgtm

**Pull request merged**